nautobot/extras/utils.py

```diff
@@ -219,6 +219,26 @@ def list_subclasses(self):
         return model_classes
 
 
+@deconstructible
+class CloudTypeModelsQuery(FeaturedQueryMixin):
+    """
+    Helper class to get ContentType models that implements CloudTypeCompatibleMixin.
+    """
+
+    def list_subclasses(self):
+        """
+        Return a list of classes that inherits from CloudTypeCompatibleMixin
+        """
+        # Avoid circular imports
+        from nautobot.cloud.models import CloudTypeCompatibleMixin
+
+        model_classes = []
+        for model_class in apps.get_models():
+            if issubclass(model_class, CloudTypeCompatibleMixin):
+                model_classes.append(model_class)
+        return model_classes
```

Comment:
Ahhhh I see how it works now

Reply:
I'm not a big fan of us adding new `FooModelsQuery` classes. Can we not instead follow the pattern used by contacts, saved views, and static groups to be supportable by a general `FeatureQuery("cloud_types")` or similar?